test/public/people.spec.ts

```diff
@@ -4,166 +4,224 @@ import { FrameContexts } from '../../src/public/constants';
 import { DOMMessageEvent } from '../../src/internal/interfaces';
 import { SdkError, ErrorCode } from '../../src/public/interfaces';
 import { people } from '../../src/public/people';
+import { Utils } from '../utils';
 
-/**
- * Test cases for selectPeople API
- */
-describe('peoplePicker', () => {
-  const mobilePlatformMock = new FramelessPostMocks();
-  const minVersionForSelectPeople = '2.0.0';
-  const originalDefaultPlatformVersion = '1.6.0';
- 
-  beforeEach(() => {
-    mobilePlatformMock.messages = [];
-
-    // Set a mock window for testing
-    _initialize(mobilePlatformMock.mockWindow);
-  });
+describe('people', () => {
+  /**
+   * Test cases for selectPeople API
+   */
+  describe('peoplePicker', () => {
+    const mobilePlatformMock = new FramelessPostMocks();
+    const minVersionForSelectPeople = '2.0.0';
+    const originalDefaultPlatformVersion = '1.6.0';
+  
+    beforeEach(() => {
+      mobilePlatformMock.messages = [];
+
+      // Set a mock window for testing
+      _initialize(mobilePlatformMock.mockWindow);
+    });
 
-  afterEach(() => {
-    // Reset the object since it's a singleton
-    if (_uninitialize) {
-      _uninitialize();
-    }
-  });
+    afterEach(() => {
```

Comment:
Looks like a lot of whitespace diffs - Intentional or something we can revert to reduce noise?

Reply:
It was intentional since I added a new describe scope to the suite. I wasn't expecting the diff to look like this though (I have a gut feeling VSO wouldn't show the diff this way). 

I'd recommend looking at the file without the diff, and focusing on the tests under "describe('openCard', ..." as that's what I added.

Reply:
I just discovered you can toggle hide whitespace in GitHub, the diff looks a lot better with it on.
![image](https://user-images.githubusercontent.com/83821800/133296316-c4eddcfb-34e4-4beb-8063-98173cb81b1f.png)